README.md

```diff
@@ -27,10 +27,17 @@ conda activate movement-env
 
 ## Overview
 
-Pose estimation tools, such as [DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and [SLEAP](https://sleap.ai/) are now commonplace when processing video data of animal behaviour. There is not yet a standardised, easy-to-use way to process the pose tracks produced from these software packages.
-
-movement aims to provide a consistent modular interface to analyse pose tracks, allowing steps such as data cleaning, visualisation and motion quantification.
-We aim to support a range of pose estimation packages, along with 2D or 3D tracking of single or multiple individuals.
+Machine learning-based tools such as
+[DeepLabCut](https://www.mackenziemathislab.org/deeplabcut) and
+[SLEAP](https://sleap.ai/) have become commonplace for tracking the
+movements of animals and their body parts in videos.
+However, there is still a need for a standardized, easy-to-use method
```

Comment:
```suggestion
However, there is still a need for standardized, easy-to-use methods
```
maybe?

Reply:
Hmm not sure about this one. I like the plural because there are many methods for processing these data, saying "a method" implies a single pipeline somehow.

Reply:
yes I agree. That is why I suggest the plural form 😁

Reply:
Good, we are agreed then.